vuepress/docs/next/docs/consume/entando-apis.md

```diff
@@ -3,139 +3,80 @@ sidebarDepth: 2
 ---
 # Accessing Entando APIs
 
-## Overview
 
-Entando includes the Swagger UI in a quickstart environment and is reachable at `/entando-de-app/api/swagger-ui.html`:
+Entando includes the Swagger UI for API access in a quickstart environment. This document presents an overview and how to enable and access the Swagger UI for Entando APIs.
 
-    http://[your-host-name]/entando-de-app/api/swagger-ui.html
+## APIs Overview
+The Entando core exposes REST APIs for every action that can be taken in the App Builder. For example, you can use APIs to create pages, page templates or add widgets to pages. The APIs can be used to support automation, testing, or integrations with external systems as well.
 
-### Enable or disable the Swagger UI in a running container
+## API Structure
+All the APIs share a common top level structure. Each response will contain a top level entry for errors, metadata, and payload.
 
-The Swagger UI can be enabled or disabled in a running container by modifying the SPRING_PROFILES_ACTIVE environment variable for the entando-de-app container. 
+The errors will always contain code and a message string indicating an error condition in the request. The metadata section is used for paging, sorting, filtering and data that is distinct from the body. The body of each response is included in the payload section of the response and varies according to each API.
 
-1. Edit the deployment. The name may be different outside of a quickstart environment. Note: Use the [ent CLI](../getting-started/entando-cli.md) to send commands to Kubernetes from the host machine.
-```
-kubectl -n entando edit deployment/quickstart-server-deployment
-```
+## Models
+All of the model classes returned by the Entando core are annotated so that the model definition is included in the Swagger documentation. At the bottom of the Swagger page all of the model classes returned by the API endpoints can be found.
```

Comment:
another aha. your explanation connects the dots... if the first sentence can be modified to include the gist of that it would be worthwhile. like "All of the model classes for the data returned by the App Engine are annotated, with the annotations defined in the Swagger documentation." this is how i read your comment, which is a bit different from - and more clear than - the 1st sentence as-is

Reply:
it's been changed already